.github/pull_request_template.md

```diff
@@ -78,21 +78,38 @@ How to retest this PR or trigger a specific build:
   Run checks against Quarkus main branch  
   Please add comment: <b>Jenkins (re)run [optaplanner|optaplanner-quickstarts|optaweb-vehicle-routing] quarkus-branch</b>
 
+- for <b>quarkus lts checks</b>  
+  Run checks against Quarkus lts branch  
+  Please add comment: <b>Jenkins run quarkus-lts</b>
+
+- for a <b>specific quarkus lts check</b>  
+  Run checks against Quarkus lts branch  
+  Please add comment: <b>Jenkins (re)run [optaplanner|optaplanner-quickstarts|optaweb-vehicle-routing] quarkus-lts</b>
+
 - for <b>native checks</b>  
   Run native checks  
   Please add comment: <b>Jenkins run native</b>
 
 - for a <b>specific native check</b>  
   Run native checks 
-  Please add comment: <b>Jenkins (re)run [optaplanner|optaplanner-quickstarts|optaweb-vehicle-routing] native</b>
+  Please add comment: <b>Jenkins (re)run [optaplanner|optaplanner-quickstarts] native</b>
 
 - for <b>mandrel checks</b>  
   Run native checks against Mandrel image
   Please add comment: <b>Jenkins run mandrel</b>
 
 - for a <b>specific mandrel check</b>  
   Run native checks against Mandrel image  
-  Please add comment: <b>Jenkins (re)run [optaplanner|optaplanner-quickstarts|optaweb-vehicle-routing] mandrel</b>
+  Please add comment: <b>Jenkins (re)run [optaplanner|optaplanner-quickstarts] mandrel</b>
+
+- for <b>mandrel lts checks</b>  
```

Comment:
I assume this runs the check for both the optaplanner and quickstarts?

Reply:
yes